### PR TITLE
delete complete

### PR DESCRIPTION
**Pull request by TsunamiCode5**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ class App extends Component {
   }
 
   componentDidMount(){
-    Promise.all(this.readNomaddaDestinations(),
-    this.readNomaddaCountries())
+    Promise.all([this.readNomaddaDestinations(),
```
Comment (triciasykes) [inline app/javascript/components/App.js:29]: Does this work?

File: app/controllers/destinations_controller.rb [hunk 2]
```diff
@@ -23,6 +23,12 @@ def create
         end
     end
 
+    def destroy
+        destination = Destination.find(params[:id])
+        destination.destroy
+        render json: destination
+    end
+
```
Comment (triciasykes) [inline app/controllers/destinations_controller.rb:31]: how do you render what you just destroyed?  :)

**Review by triciasykes — CHANGES_REQUESTED** [review]
couple of comments below

File: app/javascript/components/App.js (hunk 1 above)
Comment (TsunamiCode5) [reply app/javascript/components/App.js:29]: yes that's the only way it works in an array and having them both. Kevin helped us find that out

File: app/controllers/destinations_controller.rb (hunk 2 above)
Comment (TsunamiCode5) [reply app/controllers/destinations_controller.rb:31]: I am tracking!

Comment (triciasykes) [reply app/controllers/destinations_controller.rb:31]: you can just use 'destinations'

**Review by triciasykes — CHANGES_REQUESTED** [review]
one more comment

File: app/controllers/destinations_controller.rb (hunk 2 above)
Comment (TsunamiCode5) [reply app/controllers/destinations_controller.rb:31]: Oh yeah ! Like the first method declares

**Review by triciasykes — APPROVED** [review]

**Pull request merged**